frontend/src/views/activity/Activity.vue

```diff
@@ -156,6 +156,15 @@ export default {
     PdfDownloadButtonNuxt
   },
   mixins: [campRoleMixin],
+  provide () {
+    return {
+      contentNodeOwner: {
+        preferredContentTypes: () => this.preferredContentTypes,
+        contentNodes: () => this.contentNodes,
+        camp: () => this.camp
+      }
```

Comment:
Maybe an other name for this object could be a solution. I struggled here a bit to make the provide/inject data reactive and it finally worked when I encapsulated everything in this contentNodeOwner object.

I don't necessarily want to provide the real owner (which is a category or an activity). Otherwise the components below have to include separate logic to figure out whether the owner is a category or an activity. This way I can just inject the `preferredContentTypes` and the subcomponents don't have to care, whether they live inside a category or inside an activity.

Reply:
So far the only relevant difference between activity and category is the `preferredContentTypes`. If we added a `#[RelatedCollectionLink()] function getPreferredContentTypes() { ... }` to `Activity`, wouldn't that allow us to provide a real API entity from the store here?
Might involve the same discussion as in https://github.com/ecamp/ecamp3/pull/2700#discussion_r867438094 though.

Reply:
Improved with https://github.com/ecamp/ecamp3/pull/2683/commits/0745c227c75ac85e88d51ba517bbffcb20d1a94f